include/klee/Config/config.h

```diff
@@ -13,4 +13,4 @@
 // limitations under the License.
 
 #define LLVM_VERSION_MAJOR 3
-#define LLVM_VERSION_MINOR 6
```

Comment:
It is. My squash includes all master commits since my initial commit in my branch. I did a test merge on my local master and it applies cleanly.